phongo_compat.h

```diff
@@ -112,3 +145,76 @@
 # endif
 #endif
 
+#if PHP_VERSION_ID >= 70000
+# define phongo_char zend_string
+# define phongo_char_pdup(str) pestrdup(filename->val, 1)
```

Comment:
I bet its only used in one place, and "filename" is the variable that is passed to it